api/metricsdata.py

```diff
@@ -92,7 +92,7 @@ def get_template_data(self):
 
 class PopularityTimelineHandler(TimelineHandler):
 
-  CACHE_KEY = 'css_pop_timeline'
+  CACHE_KEY = 'metrics|css_pop_timeline'
```

Comment:
We should make `'metrics'` a constant, like the way that it is done in the Feature class.